src/arrayMethodSort.js

```diff
@@ -4,9 +4,26 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = (a, b) => `${a}` > `${b}`) {
+    for (let i = 0; i < this.length - 1; i++) {
+      if (
+        compareFunction(this[i], this[i + 1]) === true ||
+        compareFunction(this[i], this[i + 1]) > 0
+      ) {
+        const temporary = this[i];
+
+        this[i] = this[i + 1];
+
+        this[i + 1] = temporary;
+
+        i = -1;
+      }
+    }
+
+    return this;
```

Comment:
The method correctly returns the sorted array, but keep in mind that bubble sort has a worst-case and average complexity of O(n^2), where n is the number of elements. For larger arrays, a more efficient sorting algorithm like quicksort or mergesort would be more appropriate.